integration/run_test.go

```diff
@@ -19,31 +19,58 @@ limitations under the License.
 package integration
 
 import (
+	"flag"
+	"os"
 	"os/exec"
 	"testing"
 
 	"github.com/GoogleCloudPlatform/skaffold/pkg/skaffold/kubernetes"
 	"github.com/GoogleCloudPlatform/skaffold/pkg/skaffold/util"
+	"github.com/sirupsen/logrus"
 )
 
+var gkeZone = flag.String("gke-zone", "us-central1-a", "gke zone")
+var gkeClusterName = flag.String("gke-cluster-name", "integration-tests", "name of the integration test cluster")
+var gcpProject = flag.String("gcp-project", "k8s-skaffold", "the gcp project where the integration test cluster lives")
+var remote = flag.Bool("remote", false, "if true, run tests on a remote GKE cluster")
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+	if *remote {
+		cmd := exec.Command("gcloud", "container", "clusters", "get-credentials", *gkeClusterName, "--zone", *gkeZone, "--project", *gcpProject)
```

Comment:
Should we use our exec.Command helper instead?

Reply:
The helper is used on the next line below, we don't do anything special for creating the cmd, just running

Reply:
D'oh, I was thinking of the special one we have in minikube for integration tests. Nevermind.